backend/services/notificationCookie.js

```diff
@@ -0,0 +1,14 @@
+const COOKIE_NAME = 'ui-notification-bar'
+const notificationKey = ({ id, revision }) => `${id}-${revision}`
+
+const markAsDismissed = (res, { id, revision }) => {
+  const oneYear = 52 * 24 * 3600000
+  return res.cookie(COOKIE_NAME, notificationKey({ id, revision }), { maxAge: oneYear, httpOnly: true })
```

Comment:
[We erred on caution and signed and set the secure flag in production ](https://github.com/ministryofjustice/hmpps-book-video-link/blob/main/backend/routes/amendBooking/state.ts)

Wasn't sure if this was the sort of thing that might appear on a pen test or not. 

obviously you are not storing any critical data in here

Reply:
I know what you're saying but in this case I think it's fine as it only drives the viability of the notification bar.